pkg/sql/schema_changer.go

```diff
@@ -82,10 +82,10 @@ const (
 	// the cluster to converge to seeing the schema element in the DELETE_ONLY
 	// state.
 	RunningStatusDeleteOnly jobs.RunningStatus = "waiting in DELETE-ONLY"
-	// RunningStatusDeleteAndWriteOnly is for jobs that are currently waiting on
+	// RunningStatusWriteOnly is for jobs that are currently waiting on
 	// the cluster to converge to seeing the schema element in the
-	// DELETE_AND_WRITE_ONLY state.
-	RunningStatusDeleteAndWriteOnly jobs.RunningStatus = "waiting in DELETE-AND-WRITE_ONLY"
+	// WRITE_ONLY state.
+	RunningStatusWriteOnly jobs.RunningStatus = "waiting in WRITE_ONLY"
```

Comment:
@ajwerner is it OK to change these running status messages? I think so but I ask just to be sure.

Reply:
andrew said to me offline that this is fine.